dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -181,7 +182,10 @@ private static SymbolReferenceSet scanSymbolReferencesInClass(JavaClass javaClas
           }
           break;
         case Const.CONSTANT_Methodref:
-          ConstantMethodref constantMethodref = (ConstantMethodref) constant;
+          // fall through
```

Comment:
This doesn't need `// fall-through`. It's not a fall through from one case to another; it's a multi-label case. Error Prone makes you put the fall-through comment only after at least one statement in a case.